go.sh

```diff
@@ -0,0 +1,7 @@
+./riak_test -c riak_ee_dev -t repl_fs_bench3 -D'fs_num_keys=1000000' -D'fs_strategy=keylist' -D'fs_direct=0'
```

Comment:
Why are we checking this into `riak_test`? Seems like this is someone's local test script.